docs/ko/organizer-guide.md

```diff
@@ -4,7 +4,7 @@
 
 1. 구성원 변경 요청.
 2. [PEERS_GUIDELINES.md](https://github.com/mdn/translated-content/blob/main/PEERS_GUIDELINES.md) 페이지의 `Korea (ko) content — the @yari-content-ko team, which consists of:` 수정.
```

Comment:
@nschonni Some links were removed from the bottom and some were kept as is. Is there a standard?

Reply:
Only the links that start with `https://developer.mozilla.org/` needed to move to the bottom to work around the rule

Reply:
I understand. Thank you for your reply.